marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/scenarios.modeling.tsx

```diff
@@ -30,6 +31,21 @@ describe('Entity Modeling', () => {
       cy.resetTestUser();
   });
 
+  it('can navigate between explore and model tile by clicking instance count and last processed', () => {
+    cy.waitUntil(() => entityTypeTable.getEntityLastProcessed('Person')).click();
+    tiles.getExploreTile().should('exist');
+    cy.waitUntil(() => browsePage.getSelectedEntity()).should('eq', 'Person');
+    browsePage.getClearAllButton().should('exist');
+    
+    toolbar.getModelToolbarIcon().click();
```

Comment:
Since we are testing the click action on model tool icon in other places, should we test using browser back button to go back to model tile?
Also what do you think about verifying the url after navigation?
And finally can we add the click of instance/last processed to existing test in json/xml scenarios in explorer?

Reply:
I was planning on adding a model reader scenario and can start adding more navigation testing then.